lib/es6.js

```diff
@@ -22,8 +22,9 @@ function asyncPool(poolLimit, array, iteratorFn) {
     if (i === array.length) {
       return Promise.resolve();
     }
+    const currentIdx = i;
```

Comment:
Let's use the same variable name (for consistency)
```suggestion
    const index = i;
```